src/wurf/waf_resolve_context.py

```diff
@@ -159,7 +159,6 @@ def cmd_and_log(self, cmd, **kwargs):
             return super(WafResolveContext, self).cmd_and_log(
                 cmd=cmd, **kwargs)
         except WafError as e:
-            tb = traceback.format_exc()
-            raise CmdAndLogError(error=e, traceback=tb)
+            raise CmdAndLogError(error=e)
```

Comment:
No matter what command you executed, that traceback always had the same 2 lines: something useless in waf's own cmd_and_log. But we still get the meaningful traceback in the log, because waf writes it there, so we don't need to print it here.